### PR TITLE
IfcSwitchingDevice Rail Import BIMQ 2021-07-21T07:31:16.5169641Z

### PR DESCRIPTION
**Pull request by jmirtsch**
Rail properties for IfcSwitchingDevice

### CODE REVIEW

File: IFC4x3/Properties/a/AmountOfPhase_1BIplyxBLFfuml0KvHZ1y2/DocProperty.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="AmountOfPhase_1BIplyxBLFfuml0KvHZ1y2" Name="AmountOfPhase" UniqueId="4b4b3bfc-ecb5-4fa7-8c2f-014e518c1f02" PrimaryDataType="IfcInteger" />
+
```
[the remark this answers is not in the document]
Comment (BenzclyZhang) [reply IFC4x3/Properties/a/AmountOfPhase_1BIplyxBLFfuml0KvHZ1y2/DocProperty.xml]: Changed to NumberOfPhases: https://github.com/bSI-InfraRoom/IFC-Specification/pull/182/commits/9140c560d033ed13634d76a6b2712466ba5164ba

File: IFC4x3/Properties/a/AmountOfPhase_1BIplyxBLFfuml0KvHZ1y2/Documentation.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Number of phase that the equipment operates on.
```
[the remark this answers is not in the document]
Comment (BenzclyZhang) [reply IFC4x3/Properties/a/AmountOfPhase_1BIplyxBLFfuml0KvHZ1y2/Documentation.md]: Updated: https://github.com/bSI-InfraRoom/IFC-Specification/pull/182/commits/9140c560d033ed13634d76a6b2712466ba5164ba

File: IFC4x3/Properties/b/BreakingCapacity_0SRRZKplf9oAKBz3w0a2po/DocProperty.xml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="BreakingCapacity_0SRRZKplf9oAKBz3w0a2po" Name="BreakingCapacity" UniqueId="1c6db8d4-cefa-49c8-a50b-f43e80902cf2" PrimaryDataType="IfcReal" />
```
[the remark this answers is not in the document]
Comment (BenzclyZhang) [reply IFC4x3/Properties/b/BreakingCapacity_0SRRZKplf9oAKBz3w0a2po/DocProperty.xml]: Updated accordingly: https://github.com/bSI-InfraRoom/IFC-Specification/pull/182/commits/9140c560d033ed13634d76a6b2712466ba5164ba

File: IFC4x3/Properties/b/ButtonNumber_33JUHhmwL6KuSlAycJPhnq/Documentation.md [hunk 4]
```diff
@@ -0,0 +1 @@
+Identifier of the button used in the equipment panel.
```
[the remark this answers is not in the document]
Comment (BenzclyZhang) [reply IFC4x3/Properties/b/ButtonNumber_33JUHhmwL6KuSlAycJPhnq/Documentation.md]: This property is removed: https://github.com/bSI-InfraRoom/IFC-Specification/pull/182/commits/9140c560d033ed13634d76a6b2712466ba5164ba

File: IFC4x3/Properties/e/EquipmentAssociated_3dGTpqW2v9KOOy_abBbsqR/Documentation.md [hunk 5]
```diff
@@ -0,0 +1 @@
+The ID of the equipment / facility where the equipment is applied.
```
[the remark this answers is not in the document]
Comment (BenzclyZhang) [reply IFC4x3/Properties/e/EquipmentAssociated_3dGTpqW2v9KOOy_abBbsqR/Documentation.md]: Removed: https://github.com/bSI-InfraRoom/IFC-Specification/pull/182/commits/9140c560d033ed13634d76a6b2712466ba5164ba

File: IFC4x3/Properties/n/NominalCurrent_3cS2eDQYn96PF3sK0_rNda/Documentation.md [hunk 6]
```diff
@@ -0,0 +1 @@
+Value of current in accordance with which the relevant performance of a transformer operated meter is fixed (IEV ref 314-07-02)
```
[the remark this answers is not in the document]
Comment (BenzclyZhang) [reply IFC4x3/Properties/n/NominalCurrent_3cS2eDQYn96PF3sK0_rNda/Documentation.md]: Reused existing one: https://github.com/bSI-InfraRoom/IFC-Specification/pull/182/commits/9140c560d033ed13634d76a6b2712466ba5164ba

File: IFC4x3/Properties/n/NominalLength_1UfvHbdsjDbQUqxse0V82L/Documentation.md [hunk 7]
```diff
@@ -0,0 +1 @@
+Length of the product described by the manufacturer.
```
[the remark this answers is not in the document]
Comment (BenzclyZhang) [reply IFC4x3/Properties/n/NominalLength_1UfvHbdsjDbQUqxse0V82L/Documentation.md]: Reused the existing NominalLength: https://github.com/bSI-InfraRoom/IFC-Specification/pull/182/commits/9140c560d033ed13634d76a6b2712466ba5164ba

File: IFC4x3/Properties/n/NominalWidth_2P5Hkrm9P2CfJZ_JJAjbng/Documentation.md [hunk 8]
```diff
@@ -0,0 +1 @@
+Width of the box
```
[the remark this answers is not in the document]
Comment (BenzclyZhang) [reply IFC4x3/Properties/n/NominalWidth_2P5Hkrm9P2CfJZ_JJAjbng/Documentation.md]: Reused existing one and removed this one: https://github.com/bSI-InfraRoom/IFC-Specification/pull/182/commits/9140c560d033ed13634d76a6b2712466ba5164ba

File: IFC4x3/Properties/n/NominalHeight_3QVuiwSULAxvtwRDWaQCOz/Documentation.md [hunk 9]
```diff
@@ -0,0 +1 @@
+The nominal height of the object.
```
[the remark this answers is not in the document]
Comment (BenzclyZhang) [reply IFC4x3/Properties/n/NominalHeight_3QVuiwSULAxvtwRDWaQCOz/Documentation.md]: Reused existing one and removed this one: https://github.com/bSI-InfraRoom/IFC-Specification/pull/182/commits/9140c560d033ed13634d76a6b2712466ba5164ba

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_SwitchingDeviceTypeRelay/DocPropertySet.xml [hunk 10]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_SwitchingDeviceTypeRelay" UniqueId="89149be1-41bd-0973-4ca5-1c8cf3beaad6" ApplicableType="IfcSwitchingDevice/RELAY" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="NominalHeight_3QVuiwSULAxvtwRDWaQCOz" />
+		<DocProperty xsi:nil="true" href="Current_3zQJbuqOD69fAcKlNwK_SY" />
```
[the remark this answers is not in the document]
Comment (BenzclyZhang) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_SwitchingDeviceTypeRelay/DocPropertySet.xml]: This is Current, but the existing Current can be reused, so this one is removed and reused the existing one: Reused existing one and removed this one: https://github.com/bSI-InfraRoom/IFC-Specification/pull/182/commits/9140c560d033ed13634d76a6b2712466ba5164ba

File: IFC4x3/Properties/c/Current_3zQJbuqOD69fAcKlNwK_SY/Documentation.md [hunk 11]
```diff
@@ -0,0 +1 @@
+The actual current and operable range.
```
[the remark this answers is not in the document]
Comment (BenzclyZhang) [reply IFC4x3/Properties/c/Current_3zQJbuqOD69fAcKlNwK_SY/Documentation.md]: Exactly: Reused existing one and removed this one: https://github.com/bSI-InfraRoom/IFC-Specification/pull/182/commits/9140c560d033ed13634d76a6b2712466ba5164ba

File: IFC4x3/Properties/n/NumberOfEmergencyButtons_1ySlWeb$P2uBH_ablyiBv9/Documentation.md [hunk 12]
```diff
@@ -0,0 +1 @@
+The number of emergency buttons built in the system
```
[the remark this answers is not in the document]
Comment (BenzclyZhang) [reply IFC4x3/Properties/n/NumberOfEmergencyButtons_1ySlWeb$P2uBH_ablyiBv9/Documentation.md]: Change "system" to "device".

File: IFC4x3/Properties/n/NumberOfRelais_2Yk89uN9b7hOCzGWSTtXeS/DocProperty.xml [hunk 13]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NumberOfRelais_2Yk89uN9b7hOCzGWSTtXeS" Name="NumberOfRelais" UniqueId="a2b88278-5c99-47ad-833d-42071dde1a1c" PrimaryDataType="IfcCountMeasure">
+	<Documentation></Documentation>
+</DocProperty>
+
```
[the remark this answers is not in the document]
Comment (BenzclyZhang) [reply IFC4x3/Properties/n/NumberOfRelais_2Yk89uN9b7hOCzGWSTtXeS/DocProperty.xml]: Updated: Reused existing one and removed this one: https://github.com/bSI-InfraRoom/IFC-Specification/pull/182/commits/9140c560d033ed13634d76a6b2712466ba5164ba

File: IFC4x3/Properties/n/NumberfOverCurrentRelais_3_oAOIEqTAGwn2$PdOKte8/DocProperty.xml [hunk 14]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NumberfOverCurrentRelais_3_oAOIEqTAGwn2PdOKte8" Name="NumberfOverCurrentRelais" UniqueId="fec8a612-3b47-4a43-ac42-fd99d8537a08" PrimaryDataType="IfcCountMeasure" />
```
[the remark this answers is not in the document]
Comment (BenzclyZhang) [reply IFC4x3/Properties/n/NumberfOverCurrentRelais_3_oAOIEqTAGwn2$PdOKte8/DocProperty.xml]: Updated: Reused existing one and removed this one: https://github.com/bSI-InfraRoom/IFC-Specification/pull/182/commits/9140c560d033ed13634d76a6b2712466ba5164ba

File: IFC4x3/Properties/r/RatedVoltage_1O0Yqu5Eb3z8qR1wXeLRCi/Documentation.md [hunk 15]
```diff
@@ -0,0 +1 @@
+Rated value of the voltage assigned by the manufacturer to a component, device or equipment and to which operation and performance characteristics are referred.
```
[the remark this answers is not in the document]
Comment (BenzclyZhang) [reply IFC4x3/Properties/r/RatedVoltage_1O0Yqu5Eb3z8qR1wXeLRCi/Documentation.md]: Reused: Reused existing one and removed this one: https://github.com/bSI-InfraRoom/IFC-Specification/pull/182/commits/9140c560d033ed13634d76a6b2712466ba5164ba

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcElectricalDomain/PropertySets/Pset_SwitchingDeviceTypeEmergencyStop/DocPropertySet.xml [hunk 16]
```diff
@@ -9,6 +9,20 @@
...
+		<DocProperty xsi:nil="true" href="NumberOfRelais_2Yk89uN9b7hOCzGWSTtXeS" />
+		<DocProperty xsi:nil="true" href="NumberfOverCurrentRelais_3_oAOIEqTAGwn2PdOKte8" />
+		<DocProperty xsi:nil="true" href="NumberOfPoles_0PbRBXQxbBDPGvPwwmO6f" />
+		<DocProperty xsi:nil="true" href="NominalCurrent_3cS2eDQYn96PF3sK0_rNda" />
+		<DocProperty xsi:nil="true" href="RatedFrequency_0xk7ZvULjDVACUV9Qs5Q4T" />
+		<DocProperty xsi:nil="true" href="RatedVoltage_1O0Yqu5Eb3z8qR1wXeLRCi" />
+		<DocProperty xsi:nil="true" href="TransformationRatio_1OT80gqCT1y89fMoREiKYr" />
+		<DocProperty xsi:nil="true" href="TypeOfArcExtinction_1iKRDrl9j6IQuyzsIknVc2" />
 	</Properties>
 </DocPropertySet>
```
[the remark this answers is not in the document]
Comment (BenzclyZhang) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcElectricalDomain/PropertySets/Pset_SwitchingDeviceTypeEmergencyStop/DocPropertySet.xml:28]: Two Psets are removed, and all Psets have documentation now: Reused existing one and removed this one: https://github.com/bSI-InfraRoom/IFC-Specification/pull/182/commits/9140c560d033ed13634d76a6b2712466ba5164ba

File: IFC4x3/Properties/r/ReferenceEnvironmentTemperature_2vgOshA4D0$ems5HSBwemr/DocProperty.xml [hunk 17]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="ReferenceEnvironmentTemperature_2vgOshA4D0ems5HSBwemr" Name="ReferenceEnvironmentTemperature" UniqueId="b9a98dab-2843-40fe-8c36-15170bea8c35" PropertyType="p_boundedvalue" PrimaryDataType="IfcRatioMeasure" />
+
```
[the remark this answers is not in the document]
Comment (BenzclyZhang) [reply IFC4x3/Properties/r/ReferenceEnvironmentTemperature_2vgOshA4D0$ems5HSBwemr/DocProperty.xml:3]: This was certainly a mistake:). Updated: https://github.com/bSI-InfraRoom/IFC-Specification/pull/182/commits/81270448d5ae7c9983218f38928507404e580e32

File: IFC4x3/Properties/r/ReferenceEnvironmentTemperature_2vgOshA4D0$ems5HSBwemr/Documentation.md [hunk 18]
```diff
@@ -0,0 +1 @@
+Best range of temperature.
```
[the remark this answers is not in the document]
Comment (BenzclyZhang) [reply IFC4x3/Properties/r/ReferenceEnvironmentTemperature_2vgOshA4D0$ems5HSBwemr/Documentation.md:1]: Updated: https://github.com/bSI-InfraRoom/IFC-Specification/pull/182/commits/81270448d5ae7c9983218f38928507404e580e32

File: IFC4x3/Properties/t/TypeOfArcExtinction_1iKRDrl9j6IQuyzsIknVc2/Documentation.md [hunk 19]
```diff
@@ -0,0 +1 @@
+Type of arc extinction used in EmergencySwitchOff
```
[the remark this answers is not in the document]
Comment (BenzclyZhang) [reply IFC4x3/Properties/t/TypeOfArcExtinction_1iKRDrl9j6IQuyzsIknVc2/Documentation.md]: Agree on this one, so removed: https://github.com/bSI-InfraRoom/IFC-Specification/pull/182/commits/81270448d5ae7c9983218f38928507404e580e32

File: IFC4x3/Properties/t/TypeOfSelectorSwitch_2HvJM6h$z3w8yVMgXDnUfv/Documentation.md [hunk 20]
```diff
@@ -0,0 +1 @@
+Type of Switch
```
[the remark this answers is not in the document]
Comment (BenzclyZhang) [reply IFC4x3/Properties/t/TypeOfSelectorSwitch_2HvJM6h$z3w8yVMgXDnUfv/Documentation.md]: Agree also on this one, removed: https://github.com/bSI-InfraRoom/IFC-Specification/pull/182/commits/81270448d5ae7c9983218f38928507404e580e32

File: IFC4x3/PropertyConstants/c/CHANGEOVERSWITCH_0anGzhBEHCGw1gdNMuz5HO.xml [hunk 21]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="CHANGEOVERSWITCH_0anGzhBEHCGw1gdNMuz5HO" Name="CHANGEOVERSWITCH" UniqueId="24c50f6b-2ce4-4c43-a06a-9d75b8f45458" />
+
```
[the remark this answers is not in the document]
Comment (BenzclyZhang) [reply IFC4x3/PropertyConstants/c/CHANGEOVERSWITCH_0anGzhBEHCGw1gdNMuz5HO.xml]: All properties that have enumerated values are removed, so all previously added property constants are removed too: https://github.com/bSI-InfraRoom/IFC-Specification/pull/182/commits/81270448d5ae7c9983218f38928507404e580e32

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_SwitchingDeviceTypeRelay/DocPropertySet.xml [hunk 22]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_SwitchingDeviceTypeRelay" UniqueId="89149be1-41bd-0973-4ca5-1c8cf3beaad6" ApplicableType="IfcSwitchingDevice/RELAY" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="NominalHeight_1Fy4XYaJXDTBDDb9f06ofi" />
+		<DocProperty xsi:nil="true" href="Current_1I1maXboTEcP5xShYx_g0" />
```
Comment (larswik) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_SwitchingDeviceTypeRelay/DocPropertySet.xml:5]: Current or NominalCurrent?

**Review by larswik — COMMENTED** [review]
Just a check on Current vs NominalCurrent for Pset_SwitchingDeviceTypeRelay. Otherwise LFTM!

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_SwitchingDeviceTypeRelay/DocPropertySet.xml (hunk 22 above)
Comment (czapplitec) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_SwitchingDeviceTypeRelay/DocPropertySet.xml:5]: Checked again. It requires the actual working current, so Current is more proper. And WorkingVoltage is updated to Voltage https://github.com/bSI-InfraRoom/IFC-Specification/pull/182/commits/e369102bc553b4469958662ea119e1a89443022c

Comment (larswik) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_SwitchingDeviceTypeRelay/DocPropertySet.xml:5]: Ok, then ok from my side!

**Review by larswik — APPROVED** [review]

File: IFC4x3/Properties/i/InsulationResistance_1ZlGxOxPPFmwm2gfaKJrQQ/DocProperty.xml [hunk 23]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="InsulationResistance_1ZlGxOxPPFmwm2gfaKJrQQ" Name="InsulationResistance" UniqueId="63bd0ed8-ed96-4fc3-ac02-aa99144f569a" PrimaryDataType="IfcElectricResistanceMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/i/InsulationResistance_1ZlGxOxPPFmwm2gfaKJrQQ/DocProperty.xml:2]: Think I have seen this before. I hope it is the same one.

File: IFC4x3/Properties/n/NominalPower_2pu16hiH1CqQv04BNgxAph/DocProperty.xml [hunk 24]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NominalPower_2pu16hiH1CqQv04BNgxAph" Name="NominalPower" UniqueId="b3e011ab-b110-4cd1-ae40-10b5eaecaceb" PrimaryDataType="IfcPowerMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/n/NominalPower_2pu16hiH1CqQv04BNgxAph/DocProperty.xml]: Reuse?

File: IFC4x3/Properties/n/NominalPower_2pu16hiH1CqQv04BNgxAph/Documentation.md [hunk 25]
```diff
@@ -0,0 +1 @@
+A conventional value of apparent power determining a value of the rated current that may be carried with rated voltage applied, under specified conditions. ( IEV ref 421-04-04)
```
Comment (SergejMuhic) [inline IFC4x3/Properties/n/NominalPower_2pu16hiH1CqQv04BNgxAph/Documentation.md]: Just modify the existing documentation?

File: IFC4x3/Properties/n/NumberOfEarthFaultRelays_1k$ZC8XKT1wuKG_W4CR0da/Documentation.md [hunk 26]
```diff
@@ -0,0 +1 @@
+Indicates number of relays used for preventing earth fault.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/n/NumberOfEarthFaultRelays_1k$ZC8XKT1wuKG_W4CR0da/Documentation.md]:
```suggestion
Indicates the number of relays used for preventing earth fault.
```

File: IFC4x3/Properties/p/PullInVoltage_2dz9sHSqLEL9DJguaAm4sn/DocProperty.xml [hunk 27]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PullInVoltage_2dz9sHSqLEL9DJguaAm4sn" Name="PullInVoltage" UniqueId="a7f49d91-7345-4e54-9353-ab890ac04db1" PrimaryDataType="IfcElectricVoltageMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/p/PullInVoltage_2dz9sHSqLEL9DJguaAm4sn/DocProperty.xml:2]: I would definitely propose to look into a more generic solution where the pset supplies the context.

File: IFC4x3/Properties/t/TransformationRatio_1OT80gqCT1y89fMoREiKYr/Documentation.md [hunk 28]
```diff
@@ -0,0 +1 @@
+The ratio of the actual primary current or voltage to the actual secondary current or voltage
```
Comment (SergejMuhic) [inline IFC4x3/Properties/t/TransformationRatio_1OT80gqCT1y89fMoREiKYr/Documentation.md]:
```suggestion
The ratio of the actual primary current or voltage to the actual secondary current or voltage.
```

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcElectricalDomain/PropertySets/Pset_SwitchingDeviceTypeSelectorSwitch/DocPropertySet.xml [hunk 29]
```diff
@@ -11,6 +11,12 @@
...
 		<DocProperty xsi:nil="true" href="SwitchUsage_1m6hY0qWuHuO00025QrEV" />
 		<DocProperty xsi:nil="true" href="SwitchActivation_20oow0qWuHuO00025QrEV" />
+		<DocProperty xsi:nil="true" href="NominalCurrent_1Sia_y4DvB595bUBLM1Dsb" />
+		<DocProperty xsi:nil="true" href="ReferenceEnvironmentTemperature_2vgOshA4D0ems5HSBwemr" />
+		<DocProperty xsi:nil="true" href="FunctionModuleDescription_14lu9Gskf3geBmwv46Vl3" />
+		<DocProperty xsi:nil="true" href="RatedFrequency_0xk7ZvULjDVACUV9Qs5Q4T" />
+		<DocProperty xsi:nil="true" href="AmountOfPhase_1BIplyxBLFfuml0KvHZ1y2" />
+		<DocProperty xsi:nil="true" href="TypeOfSelectorSwitch_2HvJM6hz3w8yVMgXDnUfv" />
 	</Properties>
 </DocPropertySet>
```
[the remark this answers is not in the document]
Comment (czapplitec) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcElectricalDomain/PropertySets/Pset_SwitchingDeviceTypeSelectorSwitch/DocPropertySet.xml:20]: This property is removed https://github.com/bSI-InfraRoom/IFC-Specification/pull/182/commits/81270448d5ae7c9983218f38928507404e580e32

File: IFC4x3/Properties/i/InsulationResistance_1ZlGxOxPPFmwm2gfaKJrQQ/DocProperty.xml (hunk 23 above)
Comment (czapplitec) [reply IFC4x3/Properties/i/InsulationResistance_1ZlGxOxPPFmwm2gfaKJrQQ/DocProperty.xml:2]: Yes, IfcSensor also has this property. It's the same one.

File: IFC4x3/Properties/n/NominalPower_2pu16hiH1CqQv04BNgxAph/DocProperty.xml (hunk 24 above)
Comment (czapplitec) [reply IFC4x3/Properties/n/NominalPower_2pu16hiH1CqQv04BNgxAph/DocProperty.xml]: This property has been merged. This has the IEV defintion. It is not about power consumption.

File: IFC4x3/Properties/n/NominalPower_2pu16hiH1CqQv04BNgxAph/Documentation.md (hunk 25 above)
Comment (czapplitec) [reply IFC4x3/Properties/n/NominalPower_2pu16hiH1CqQv04BNgxAph/Documentation.md]: This one is merged already

**Review by larswik — APPROVED** [review]

File: IFC4x3/Properties/p/PullInVoltage_2dz9sHSqLEL9DJguaAm4sn/DocProperty.xml (hunk 27 above)
Comment (czapplitec) [reply IFC4x3/Properties/p/PullInVoltage_2dz9sHSqLEL9DJguaAm4sn/DocProperty.xml:2]: This is a specific term associated to relay. It is not the same with RatedVoltage or Voltage (working voltage) in IFC. Some reference here: https://www3.panasonic.biz/ac/e/control/relay/term/index.jsp

**Pull request merged**